src/containers/BooksForm.js

```diff
@@ -1,22 +1,50 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { createBook } from '../actions/index';
 
 const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
 
 const renderCategories = (categories) => categories.map((category) => (
   <option value={category} key={category}>{category}</option>
 ));
-console.log(renderCategories(categories));
-const BookForm = () => (
-  <div>
-    <input type="text" id="title" placeholder="Title" />
-    <label htmlFor="categories">
-      Category:
-      <select id="categories">
-        {renderCategories(categories)}
-      </select>
-      <button type="submit">Add Book</button>
-    </label>
-  </div>
-);
+
+const BookForm = () => {
+  const [state, setState] = useState({ title: '', category: 'Action' });
+  const invoke = useDispatch();
+  const updateTitle = (e) => {
+    const title = e.target.value;
+    setState((prevState) => ({ ...prevState, title }));
+  };
+
+  const updateCategory = (e) => {
+    const category = e.target.value;
+    setState((prevState) => ({ ...prevState, category }));
+  };
```

Comment:
- [ ] [REQUIRED] Instead of having these, you should implement `handleChange` method and attache it to input's and select's onChange event 
[screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleChange.png)